cmd/bootcfg/main.go

```diff
@@ -13,38 +14,60 @@ import (
 	"github.com/coreos/pkg/flagutil"
 )
 
-var log = capnslog.NewPackageLogger("github.com/coreos/coreos-baremetal/cmd/bootcfg", "main")
+var (
+	// version provided by compile time flag: -ldflags "-X main.version $GIT_SHA"
+	version string
```

Comment:
Added the message, thanks. Also, the anonymous struct flags is a nice touch I've adopted it :+1: